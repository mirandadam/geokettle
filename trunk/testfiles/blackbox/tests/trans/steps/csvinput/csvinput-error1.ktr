<?xml version="1.0" encoding="UTF-8"?>
<transformation>
  <info>
    <name>csv test 3</name>
    <description/>
    <extended_description/>
    <trans_version/>
    <directory>&#47;</directory>
    <log>
      <read/>
      <write/>
      <input/>
      <output/>
      <update/>
      <rejected/>
      <connection/>
      <table/>
      <use_batchid>Y</use_batchid>
      <use_logfield>N</use_logfield>
    </log>
    <maxdate>
      <connection/>
      <table/>
      <field/>
      <offset>0.0</offset>
      <maxdiff>0.0</maxdiff>
    </maxdate>
    <size_rowset>1000</size_rowset>
    <sleep_time_empty>100</sleep_time_empty>
    <sleep_time_full>100</sleep_time_full>
    <unique_connections>N</unique_connections>
    <feedback_shown>Y</feedback_shown>
    <feedback_size>5000</feedback_size>
    <using_thread_priorities>N</using_thread_priorities>
    <shared_objects_file/>
    <dependencies>
    </dependencies>
    <partitionschemas>
    </partitionschemas>
    <slaveservers>
    </slaveservers>
    <clusterschemas>
    </clusterschemas>
  <modified_user>-</modified_user>
  <modified_date>2007&#47;07&#47;24 22:33:59.679</modified_date>
  </info>
  <notepads>
  </notepads>
  <connection>
    <name>Pentaho</name>
    <server>localhost</server>
    <type>MYSQL</type>
    <access>Native</access>
    <database>MBO</database>
    <port>3306</port>
    <username>root</username>
    <password>Encrypted </password>
    <servername/>
    <data_tablespace/>
    <index_tablespace/>
    <attributes>
      <attribute><code>EXTRA_OPTION_MYSQL.defaultFetchSize</code><attribute>500</attribute></attribute>
      <attribute><code>EXTRA_OPTION_MYSQL.useCursorFetch</code><attribute>true</attribute></attribute>
      <attribute><code>IS_CLUSTERED</code><attribute>N</attribute></attribute>
      <attribute><code>MAXIMUM_POOL_SIZE</code><attribute>10</attribute></attribute>
      <attribute><code>PORT_NUMBER</code><attribute>3306</attribute></attribute>
      <attribute><code>STREAM_RESULTS</code><attribute>Y</attribute></attribute>
      <attribute><code>USE_POOLING</code><attribute>N</attribute></attribute>
    </attributes>
  </connection>
  <order>
  </order>
  <step>
    <name>datafile</name>
    <type>CsvInput</type>
    <description/>
    <distribute>Y</distribute>
    <copies>1</copies>
         <partitioning>
           <method>none</method>
           <field_name/>
           <schema_name/>
           </partitioning>
    <filename>ftp:&#47;&#47;bogus&#47;testfiles&#47;customer-dimension-test.csv</filename>
    <separator>;</separator>
    <enclosure>&quot;</enclosure>
    <header>Y</header>
    <buffer_size>30</buffer_size>
    <lazy_conversion>Y</lazy_conversion>
    <fields>
      <field>
        <name>date1</name>
        <type>Date</type>
        <format>dd-MM-yyyy</format>
        <currency/>
        <decimal/>
        <group/>
        <length>-1</length>
        <precision>-1</precision>
        </field>
      <field>
        <name>Customer_No</name>
        <type>Integer</type>
        <format>#</format>
        <currency/>
        <decimal/>
        <group/>
        <length>5</length>
        <precision>0</precision>
        </field>
      <field>
        <name>First_Name</name>
        <type>String</type>
        <format/>
        <currency/>
        <decimal/>
        <group/>
        <length>-1</length>
        <precision>-1</precision>
        </field>
      <field>
        <name>Last_Name</name>
        <type>String</type>
        <format/>
        <currency/>
        <decimal/>
        <group/>
        <length>-1</length>
        <precision>-1</precision>
        </field>
      <field>
        <name>Address</name>
        <type>String</type>
        <format/>
        <currency/>
        <decimal/>
        <group/>
        <length>-1</length>
        <precision>-1</precision>
        </field>
      <field>
        <name>Birth_Date</name>
        <type>Date</type>
        <format>dd-MM-yyyy</format>
        <currency/>
        <decimal/>
        <group/>
        <length>-1</length>
        <precision>-1</precision>
        </field>
      </fields>
     <cluster_schema/>
 <remotesteps>   <input>   </input>   <output>   </output> </remotesteps>    <GUI>
      <xloc>127</xloc>
      <yloc>112</yloc>
      <draw>Y</draw>
      </GUI>
    </step>

  <step_error_handling>
  </step_error_handling>
</transformation>
