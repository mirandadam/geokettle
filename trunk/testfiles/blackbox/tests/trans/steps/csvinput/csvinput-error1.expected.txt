datafile.0 : Error opening file 'ftp://bogus/testfiles/customer-dimension-test.csv' : org.apache.commons.vfs.FileSystemException: Could not connect to FTP server on "bogus".
datafile.0 : org.apache.commons.vfs.FileSystemException: Could not connect to FTP server on "bogus".
datafile.0 : Caused by: java.net.UnknownHostException: bogus
datafile : Error initializing step [datafile]
csv test 3 : Step [datafile.0] failed to initialize!
csv test 3 : We failed to initialize at least one step.  Execution can not begin!
BlackBoxTest : Unable to prepare and initialize this transformation: testfiles/blackbox/tests/trans/steps/csvinput/csvinput-error1.ktr
